tools/tab_checker

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+pr_id=$1
+files=$(curl https://api.github.com/repos/os-autoinst/scripts/pulls/${pr_id}/files | jq -c '.[].filename' | sed 's/\"//g')
+checker() {
+for n in "$files"; do
+    if head -1 "$n" | grep -q '^#!'; then
```

Comment:
I am not sure if this is enough for the "on bash scripts" part. This will also match other scripts. Maybe use `file`?

Reply:
right. In https://github.com/os-autoinst/openQA/blob/master/tools/perlfiles#L3 we are using `file --mime-type`